src/js/bootstrap-slider.js

```diff
@@ -973,14 +997,20 @@ const windowIsDefined = (typeof window === "object");
 			- Any method that is not part of the public interface.
 			- Place it underneath this comment block and write its signature like so:
 
-			  					_fnName : function() {...}
+				_fnName : function() {...}
 
 			********************************/
 			_removeSliderEventHandlers: function() {
 				// Remove keydown event listeners
 				this.handle1.removeEventListener("keydown", this.handle1Keydown, false);
 				this.handle2.removeEventListener("keydown", this.handle2Keydown, false);
 
+				if(this.options.ticks_tooltip){
+					$('.slider-tick-container div').each(function(){
```

Comment:
Could you also add a comment above this section indicating that you are removing the hover event listeners from the ticks?